assets/src/components/v2/pre_fare_single_screen_alert.tsx

```diff
@@ -177,28 +180,33 @@ const FallbackLayout: React.ComponentType<FallbackLayoutProps> = ({
 
   const icon =
     effect === "delay" ? (
-      <ClockIcon className="alert-card__pio-text__icon" />
+      <ClockIcon className="alert-card__fallback__icon" />
     ) : effect === "shuttle" ? (
-      <ShuttleBusIcon className="alert-card__pio-text__icon" />
+      <ShuttleBusIcon className="alert-card__fallback__icon" />
     ) : (
-      <NoServiceIcon className="alert-card__pio-text__icon" />
+      <NoServiceIcon className="alert-card__fallback__icon" />
     );
 
   return (
-    <div className="alert-card__pio-text">
+    <div className="alert-card__fallback">
       {icon}
-      {issue && <div className="alert-card__pio-text__main-text">{issue}</div>}
+      {issue && <div className="alert-card__fallback__issue-text">{issue}</div>}
       {remedy && (
```

Comment:
I know you didn't touch this so not blocking, but is this necessary considering `remedy` is not optional? Wondering if it can be simplified a bit since this condition should always be true.

Reply:
Nevermind. After looking at the `WidgetInstance`, I remembered that it can be `null` and that's what the conditional is for. My bad.